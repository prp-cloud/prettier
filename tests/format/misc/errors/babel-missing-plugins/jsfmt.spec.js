runFormatTest(
  {
    importMeta: import.meta,
    snippets: [
      // https://github.com/babel/babel/commit/a466f9c310ace91484d4087f077ee6d6c8cd8789
      "export type Foo = number;",
    ],
  },
  ["babel"],
);
