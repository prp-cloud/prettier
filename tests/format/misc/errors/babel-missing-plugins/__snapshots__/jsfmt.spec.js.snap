// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snippet: #0 [babel] format 1`] = `
"Unexpected token. (1:8)
> 1 | export type Foo = number;
    |        ^"
`;
